doc/source/_toc.yml

```diff
@@ -62,7 +62,7 @@ parts:
       - file: serve/index
         title: Ray Serve
         sections:
-          - file: serve/end_to_end_tutorial
+          - file: serve/getting_started
```

Comment:
@maxpumperla do you have suggestion on how to keep alias so previously linked pages at serve/end_to_end_tutorial.html won't 404? Is it suggested for us to create a stub page?